landoapi/spec/swagger.yml

```diff
@@ -111,8 +111,70 @@ paths:
       responses:
         202:
           description: OK
+          schema:
+            type: object
+            properties:
+              id:
+                type: integer
+                description: |
+                  A newly created Landing id
         404:
-          description: Revision does not exist
+          description:
+            Revision does not exist or its request_id doesn't correspond to
+            the saved one
```

Comment:
Probably best not to mention internal implementation details here.  I don't see a `request_id` mentioned in the POST parameters, either?